packages/bvaughn-architecture-demo/src/suspense/PointsCache.ts

```diff
@@ -73,6 +98,31 @@ export function getHitPointsForLocation(
   }
 }
 
+// Wrapper method around Suspense method.
+// This method can be used by non-React code to prefetch/prime the Suspense cache by loading preview data.
+// Loaded properties can also be accessed via getCachedHitPointsForLocation().
+export async function getHitPointsForLocationHelper(
+  client: ReplayClientInterface,
+  location: Location,
+  condition: string | null,
+  focusRange: TimeStampedPointRange | null
+): Promise<HitPointsAndStatusTuple> {
+  try {
+    return getHitPointsForLocation(client, location, condition, focusRange);
+  } catch (errorOrPromise) {
+    if (
+      errorOrPromise != null &&
+      typeof errorOrPromise === "object" &&
+      typeof errorOrPromise.hasOwnProperty("then")
```

Comment:
👀 the condition will be true with any object even when there is no `then` own property because `typeof` operator always returns a string
```suggestion
      errorOrPromise.hasOwnProperty("then")
```

Reply:
Nice catch! Thanks :)